### PR TITLE
User Id in Vehicle Submission

### PR DESCRIPTION
**Pull request by rmuehler**

### CODE REVIEW

**Review by ammitchky — APPROVED** [review]
looks good

**Review by GannonDeWalt — APPROVED** [review]
Looks good

**Review by wangscs — APPROVED** [review]
nice!

**Review by 100xlongx — APPROVED** [review]
wow!!

File: VehicleTests/E2E-Tests/Vehicle/Vehicle_POST_Route.cs [hunk 1]
```diff
@@ -33,6 +33,31 @@ public class Vehicle_POST_Route
                 services.AddDbContext<DatabaseContext>(options => options.UseInMemoryDatabase("VehiclePost"));
             });
 
+        private class TokenHolder
```
Comment (mosdevly) [inline VehicleTests/E2E-Tests/Vehicle/Vehicle_POST_Route.cs:36]: somethings wrong

**Pull request merged**